app/models/track.rb

```diff
@@ -90,7 +90,7 @@ def highlightjs_language
   end
 
   def average_test_duration
-    git.average_test_duration + INFRASTRUCTURE_DURATION_S
+    git.average_test_duration.round + INFRASTRUCTURE_DURATION_S
```

Comment:
I've opted for rounding this value, as showing a test runner to run for `5.6` seconds gives a false impression of precision I feel.